hack/verify-structured-logging.sh

```diff
@@ -48,15 +48,48 @@ while IFS='' read -r line; do
   migrated_packages+=("$KUBE_ROOT/$line")
 done < <(cat "${migrated_packages_file}")
 
-
+echo "Running structured logging static check on migrated packages"
 ret=0
 GOOS=linux    logcheck "${migrated_packages[@]}" || ret=$?
 GOOS=windows  logcheck "${migrated_packages[@]}" || ret=$?
 
 if [ $ret -eq 0 ]; then
-  echo "Structured logging static check is passed :)."
+  echo "Structured logging static check passed on migrated packages :)"
 else
   echo "Please fix above failures. You can locally test via:"
   echo "hack/verify-structured-logging.sh"
 fi
+
+# Ignore migrated packages as they are already tested
+# Trim the trailing /... from given packages
+ignore_packages=`cat ${KUBE_ROOT}/hack/.structured_logging | grep -oE '^[a-zA-Z0-9/]+[^/\.]'`
+
+all_packages=()
```

Comment:
can't we just do `go list ./... ./vendor/k8s.io/...` and produce the same results?

Reply:
`go list` fails because it can not load `staging`.  So, we won't get a list to run logcheck.
So we need to skip "staging" and other problematic packages (if any) before `go list`.

`grep -v` won't work as well because it needs a list input.